src/views/v2/tickets/item.php

```diff
@@ -14,34 +14,39 @@
  *
  * @version TBD
  *
- * @var Tribe__Tickets__Ticket_Object      $ticket      The Ticket Object
- * @var WP_Post|int                        $post_id     The post object or ID.
- * @var Tribe__Tickets__Tickets            $provider    The tickets provider class.
- * @var string                             $provider_id The tickets provider class name.
- * @var Tribe__Tickets__Commerce__Currency $currency    The Currency Object
- * @var int                                $key         The ticket key.
- * @var bool                               $is_mini     True if it's in mini cart context.
- * @var bool                               $is_modal    True if it's in modal context.
+ * @var Tribe__Tickets__Ticket_Object[] $active_rsvps    The RSVPs.
+ * @var Tribe__Tickets__Tickets_Handler $tickets_handler The Tickets Handler instance.
+ * @var Tribe__Tickets__Ticket_Object   $ticket          The Ticket Object.
+ * @var WP_Post|int                     $post_id         The post object or ID.
+ * @var Tribe__Tickets__Tickets         $provider        The tickets provider class.
+ * @var string                          $provider_id     The tickets provider class name.
+ * @var int                             $key             The ticket key.
+ * @var bool                            $has_shared_cap  True if ticket has shared capacity.
+ * @var bool                            $is_mini         True if it's in mini cart context.
+ * @var bool                            $is_modal        True if it's in modal context.
  */
 
-/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
-$tickets_handler = tribe( 'tickets.handler' );
-
-$context = [
-	'ticket'          => $ticket,
-	'key'             => $this->get( 'key' ),
-	'currency_symbol' => $currency->get_currency_symbol( $ticket->ID, true ),
-];
-
 if (
 	empty( $provider )
 	|| $ticket->provider_class !== $provider->class_name
 ) {
 	return false;
 }
 
-$has_shared_cap = $tickets_handler->has_shared_capacity( $ticket );
-$has_suffix     = ! empty( $ticket->price_suffix );
+/* @var Tribe__Tickets__Privacy $privacy */
```

Comment:
Moving it from one template to another isn't what we're after, we're looking to move these sorts of things into the top level and pass them down.